chart/templates/scheduler/scheduler-deployment.yaml

```
@@ -53,7 +53,7 @@ metadata:
     release: {{ .Release.Name }}
     chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
     heritage: {{ .Release.Service }}
-    executor: {{ .Values.executor }}
+    executor: {{ trunc -63 .Values.executor }}  # 3rd-party executor path can be too long
```

```suggestion
    executor: {{ .Values.executor }}
```

No longer need to truncate right?

---

Changed.